### ISSUE TITLE
DEF FN using D accessing an array gives an syntax error

### ISSUE THREAD

**Issue by Gud305** [MEGA65/mega-basic64#5]
Disclamer: my conclusion can be wrong.

When trying the old Star Trek basic game, I discovered a wired syntax error, that seems to only appear in BASIC65, when a function named D access an array.
The following code gives a syntax error in line 60
10 DIM X(8)
20 X(2)=2
30 DEF FND(Q)=X(2)*2
40 DEF FNDM(Q)=X(2)*3
50 PRINT FNDM(0)
60 PRINT FND(0)

When running in 64 mode, it returns the correct value, without a syntax error.

**Comment by Gud305** [comment]
I tested this on V920349

**Comment by lgblgblgb** [comment]
Shouldn't be this issue at https://github.com/MEGA65/mega65-rom-public/issues instead?
@Edilbert ?

**Comment by Edilbert** [comment]
Yes, it's in the wrong repo, thank you @Gud305 for raising the issue and @lgblgblgb for bringing it to my attention. 
The issue is fixed in ROM version 920360, available in #closed-rom at discord and on the filehost.
@Gud305 , close this issue please, I have not the permission to do that.

**Comment by Gud305** [comment]
Sorry for the wrong repo. I have teste 920360, and now it can run http://vintage-basic.net/bcg/superstartrek.bas without problems
